PetAdoption-iOS/PetAdoption-iOS/AppDelegate.swift

```diff
@@ -57,10 +57,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		
 		navBar.barStyle = .black
 		
-		UIButton.appearance().tintColor = UIColor.themeTintColor()
```

Comment:
Had to comment this line out because it causes tableviewCell checkmark accessory views and button labels to be white on white (and therefore invisible). 

I tried to solve this by setting the tint color of the table view and its cells locally, but that didn't work, so for now I just commented this line out. 
Commenting this out, as a temporary solution, didn't seem to have any bad affect on the existing UI.

Reply:
I would say just go ahead and remove the line instead.  I'm not sure why we had UIButton appearances set to white.  UIBarButtonItem appearances is fine to be white, since the navigation bar is not white.